src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx

```diff
@@ -46,9 +49,16 @@ const ResidentPage: NextPage<WithUser<ResidentPageProps>> = ({
           {resident.name} | Document Evidence Service | Hackney Council
         </title>
       </Head>
-      <h1 className="lbh-heading-h2">{resident.name}</h1>
-      <p className="lbh-body">{resident.phoneNumber}</p>
-      <p className="lbh-body">{resident.email}</p>
+      <h1 className="lbh-heading-h2">
+        <Link href={`/teams/${teamId}/dashboard`}>
+          <a data-cy="search-page" className="lbh-link">
```

Comment:
I think having the `className` attribute first makes it easier to read. Also, for consistency, it would be good to change `data-cy` to `data-testid`.